docs/reference/installation.rst

```diff
@@ -157,106 +162,86 @@ Next config each of the bundles:
                         # ...
                         currency: Sonata\Component\Currency\CurrencyDoctrineType
 
-            services:
-                # Register dedicated Product Managers
-        #       sonata.product.manager.amazon:
-        #           class: Sonata\ProductBundle\Entity\ProductManager
-        #           arguments:
-        #               - Application\Sonata\ProductBundle\Entity\Amazon
-        #               - "@sonata.product.entity_manager"
+Extending the Bundle
+--------------------
+At this point, the bundle is functional, but not quite ready yet. You need
+to generate the correct entities for all bundles:
 
-        #       sonata.product.manager.bottle:
-        #           class: Sonata\ProductBundle\Entity\ProductManager
-        #           arguments:
-        #               - Application\Sonata\ProductBundle\Entity\Bottle
-        #               - "@sonata.product.entity_manager"
+.. code-block:: bash
 
-                # Register dedicated Product Providers
-        #       sonata.product.type.amazon:
-        #           class: Application\Sonata\ProductBundle\Entity\AmazonProductProvider
+    bin/console sonata:easy-extends:generate SonataBasketBundle --dest=src --namespace_prefix=App
+    bin/console sonata:easy-extends:generate SonataCustomerBundle --dest=src --namespace_prefix=App
+    bin/console sonata:easy-extends:generate SonataInvoiceBundle --dest=src --namespace_prefix=App
+    bin/console sonata:easy-extends:generate SonataOrderBundle --dest=src --namespace_prefix=App
+    bin/console sonata:easy-extends:generate SonataPaymentBundle --dest=src --namespace_prefix=App
+    bin/console sonata:easy-extends:generate SonataProductBundle --dest=src --namespace_prefix=App
 
-        #       sonata.product.type.bottle:
-        #           class: Application\Sonata\ProductBundle\Entity\BottleProductProvider
+.. note::
+    If you are not using Symfony Flex, use command without ``--namespace_prefix=App``.
 
+With provided parameters, the files are generated in ``src/Sonata``.
```

Comment:
I haven't tested all the changes yet, but a quick glance seems the files are still created in the `src/Application/Sonata` directory, with the namspace `App\Application` instead of just `App`

Reply:
Should be fixed with https://github.com/sonata-project/SonataEasyExtendsBundle/pull/209